spec/jobs/create_linked_topic_spec.rb

```diff
@@ -44,7 +44,6 @@
       new_topic = Topic.last
       linked_topic = new_topic.linked_topic
       expect(topic.title).to include(I18n.t("create_linked_topic.topic_title_with_sequence", topic_title: raw_title, count: 1))
-      expect(topic.posts.last.raw).to eq(I18n.t('create_linked_topic.moderator_post_raw', new_url: new_topic.url))
```

Comment:
Should there be a different expectation instead?

Reply:
Added it back. :)